challenges-7-plus.yml

```diff
@@ -0,0 +1,277 @@
+preview:
+  question: |
+    Hey! I've prepared this template that needs NO preview from Turbo when it loads:
+    
+    ```twig
+    {% extends 'base.html.twig' %}
+    
+    {% block title%}Play Pong in the Browser for Free!{% endblock %}
+    
+    {% block metas %}
+    
+    {% endblock %}
+    
+    {% block body %}
+    <div id="pong_game_app"></div>
+    {% endblock %}
+    ```
+    
+    I even added a `block metas` so that I can add the proper `head` tag to avoid this... but,
```

Comment:
But now... I want to avoid showing a "preview" if the user navigates back to this page. I've added
a `block metas` so that I an add the proper tag to the `head` element to avoid the preview... but